code/game/machinery/doors/airlock.dm

```diff
@@ -226,21 +226,19 @@ About the new airlock wires panel:
 /obj/machinery/door/airlock/proc/loseMainPower()
 	main_power_lost_until = mainPowerCablesCut() ? -1 : world.time + SecondsToTicks(60)
 	if(main_power_lost_until > 0)
-		main_power_timer = addtimer(src, "regainMainPower", SecondsToTicks(60), 1)
-
+		main_power_timer = addtimer(CALLBACK(src, .proc/regainMainPower), SecondsToTicks(60), TIMER_UNIQUE | TIMER_STOPPABLE)
```

Comment:
you can utilize `TIMER_UNIQUE` and `TIMER_STOPPABLE` together, correct, @MrStonedOne?

Reply:
I have no idea how these callbacks work-- does .proc/procName call the proc from the current object?

Reply:
If you're utilizing a proc that is only on the parent, but not the child, then you'd have to type in the path of the parent that has the proc. 

Thankfully, the compiler will automatically fail if it can't find the proc in question.

Reply:
Thanks for the response, makes more sense now

Reply:
yes